packages/back-end/src/controllers/datasources.ts

```diff
@@ -625,7 +625,8 @@ export async function postGoogleOauthRedirect(
   const context = getContextFromReq(req);
   const { projects } = req.body;
 
-  if (!context.permissions.canCreateDataSource({ projects })) {
+  // Huh, why? -- rb
```

Comment:
Extraneous comment?

Reply:
Well I'm not sure why that call is inside the oauth code but I'm guessing it's a smokescreen type of thing.